ui-v2/tests/unit/utils/get-object-pool-test.js

```diff
@@ -0,0 +1,98 @@
+import getObjectPool from 'consul-ui/utils/get-object-pool';
+import { module, skip } from 'qunit';
+import test from 'ember-sinon-qunit/test-support/test';
+
+module('Unit | Utility | get object pool');
+
+skip('Decide what to do if you add 2 objects with the same id');
+test('acquire adds objects', function(assert) {
+  const actual = [];
+  const expected = {
+    hi: 'there',
+    id: 'hi-there-123',
+  };
+  const expected2 = {
+    hi: 'there',
+    id: 'hi-there-456',
+  };
+  const pool = getObjectPool(function() {}, 10, actual);
+  pool.acquire(expected, expected.id);
+  assert.deepEqual(actual[0], expected);
+  pool.acquire(expected2, expected2);
```

Comment:
You aren't passing in `expected2.id` as the second argument.

Reply:
Doh, good catch!